### DISCUSSION TITLE
Osu fails to open at all on Fedora 37

### DISCUSSION THREAD

**Discussion by CoffeeIsLife87** [ppy/osu#21414] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

`Failed to create CoreCLR, HRESULT: 0x8007054F` is the error it gives me when I run it in a terminal
No window ever shows and nothing appears in `dmesg` (some other thread said to check it so I figured I'd mention it)
No log files appeared in the location mentions in https://github.com/ppy/osu/discussions/15188
Lemme know if there's anything else I should check for diagnosing the issue

### Version

2022.1123.0

### Logs

There were no log files in the listed locations

**Comment by CoffeeIsLife87** [comment, marked as answer]
I just tried using a standard kernel and it ran fine. Is there some kernel module that I need to have?
I'm currently using a custom kernel cause that's the only thing that works with my laptop speakers

**Reply by smoogipoo** [reply]
I think you're going to be on your own here. This is a .NET level issue for you which we're unable to debug. You may be able to `strace` to see where it dies, but it's up to you to figure it out from there.

**Reply by CoffeeIsLife87** [reply]
I built the kernel from source again (6.1.0-rc6+) with the patch for my sound and nothing else and now it works fine for some reason? I don't know if an update to something else fixed it but for whatever reason it works now
Sorry for the inconvenience!!
